docs/high/file.rst

```diff
@@ -132,6 +132,15 @@ compatible.
 
 The default is "earliest".
 
+Starting with HDF5 version 1.10.2, it is possible to specify another version:
+
+    >>> f = h5py.File('name.hdf5', libver='v18')
+
+that will store objects compatible with HDF5 1.8. For example, to ensure full
```

Comment:
What happens if you do this and then try to use the new 1.10 features, e.g. creating a virtual dataset? Does the 'best effort' qualifier for `'earliest'` also apply to this,  so it will break 1.8 compatibility if you need a 1.10 feature? Or will it refuse to do anything that makes the file incompatible with 1.8? And if so, how clear is the error?

Reply:
It will do the latter and throw an error. I am not sure on the error message clarity. Will have to try.

Good overview is at https://www.hdfgroup.org/2018/04/why-should-i-care-about-the-hdf5-1-10-2-release/. More technical details are at https://www.hdfgroup.org/wp-content/uploads/2018/04/RFC-Setting-Bounds-for-Object-Creation-in-HDF5-1.10.0-Update.pdf.